### ISSUE TITLE
cmdLineTester_dumpromclasstests needs to create core with -Xshareclasses:none

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#3957]
https://ci.eclipse.org/openj9/job/PullRequest-Extended-JDK8-linux_x86-64_cmprssptrs-OpenJ9/73
```
Testing: Run !dumpallromclasslinear 2 - with BCI agent core
Test start time: 2018/12/06 18:29:54 Coordinated Universal Time
Running command: /home/jenkins/workspace/PullRequest-Extended-JDK8-linux_x86-64_cmprssptrs-OpenJ9/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/bin/../../bin/jdmpview -core j9core.dmp
Time spent starting: 2 milliseconds
JVMDUMP039I Processing dump event "systhrow", detail "java/lang/OutOfMemoryError" at 2018/12/06 18:30:45 - please wait.
```

The cmdLineTester_dumpromclasstests creates a core file and then does a linear dump of all the ROM classes in that core file. Since shared classes by default is enabled, the shared cache is part of the core. There are so many ROM classes in the core file that `!dumpallromclasslinear 2` creates a massive amount of output, which often results in a OOM. 

The fix is to run the `Create core file` step with -Xshareclasses:none in order to reduce the size of the output and avoid OOM, and unexpected keywords in the output (as found in #3770)

**Comment by pshipton** [comment]
@llxia @smlambert

**Comment by pshipton** [comment]
Shelley suggested that https://github.com/eclipse/openj9/pull/3950 may resolve the problem.

**Comment by pshipton** [comment]
Seems to be resolved by #3950
